src/components/ebay-pagination/README.md

```diff
@@ -46,6 +45,10 @@ Event | Data | Description
 Name | Type | Stateful | Description
 --- | --- | --- | ---
 `disabled` | Boolean | No | Previous/next button is disabled or not
-`href` | String | No | for link that looks like a menu-item
+`href` | String | No | for link that looks like a menu-item, by omitting the item will switch to a button
```

Comment:
Looks like this description may have gotten mangled. Maybe:
`for links; omitting the href will switch to using a button`

Reply:
Ah good catch.